### ISSUE TITLE
CSS examples: example needed for list-style-type

### ISSUE THREAD

**Issue by wbamberg** [mdn/interactive-examples#483]
https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type

It should be possible to reuse most of the existing example (https://screenshots.firefox.com/lShH53qBOC6sS6eU/developer.mozilla.org).

We'd need to give some thought to the `@counter-style` option. If we want to include it, we could perhaps move the *"custom-emoji" Assume the following rule: ...* bit into the output pane of the example, and maybe reword it a little:

> With `custom-emoji` selected, the following `@counter-style` rule is applied: ...

**Comment by brianlmacdonald** [comment]
Dibs?

**Comment by wbamberg** [comment]
Thanks for offering to help out, @brianlmacdonald!

@helmutgranda is already working on `list-style` over in https://github.com/mdn/interactive-examples/pull/547, so he might want to do the other `list-style-*` properties when he's done with that (even if he doesn't, it would be nice to use the same basic example, for consistency).

If there's aren't any open other bugs here that you want to take, please feel free to take a look at the [big spreadsheet](https://docs.google.com/spreadsheets/d/1Qi1MHvx--6QGGFdboi4VaGVnKf0-kqxwOGMvfdAFhs0/edit#gid=883382855) - any items that are empty or have "Not started" in them are fair game, except for the Grid ones, which I think we will tackle separately, and except for any that are currently the [subject of open PRs](https://github.com/mdn/interactive-examples/pulls), such as `column` and `perspective`.

**Comment by brianlmacdonald** [comment]
@wbamberg can I open a new issue for cursor and tackle the css examples?
